eZ/Bundle/EzPublishCoreBundle/Resources/config/image.yml

```diff
@@ -90,11 +94,30 @@ services:
             - "@liip_imagine.filter.configuration"
             - "@?logger"
 
+    ezpublish.image_alias.imagine.alias_generator.placeholder:
+        class: 'eZ\Bundle\EzPublishCoreBundle\Imagine\PlaceholderAliasGenerator'
+        arguments:
+            - '@ezpublish.image_alias.imagine.alias_generator.inner'
```

Comment:
Would it maybe be a good opportunity to add proper aliases for those services, and use autowiring ?